apps/docs/content/components/concepts/layout.mdx

```diff
@@ -1,48 +1,46 @@
 ---
 title: Layout
-description: This page describes how to build application layouts with Hopper using Flex or Grid.
+description: Learn how to build application layouts with Hopper using Flex or Grid.
 order: 1
 ---
 
 ## Introduction
 
-Hopper includes layout components to help build applications more easily. The [Flex](./Flex) and [Grid](./Grid)
-components are containers, which are responsible for the layout of their children. `Flex` follows the
+Hopper includes layout components to help build applications. The [Flex](./Flex) and [Grid](./Grid)
+components are containers, they are responsible for the layout of their children. `Flex` follows the
 [CSS flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox)
 algorithm, while `Grid` implements
 [CSS grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout).
 
-These components provide props with pre-defined Hopper tokens for sizing, spacing,
-and other layout options. You can use `Flex` and `Grid` together as needed to build different
+These components provide props with predefined Hopper tokens for sizing, spacing,
+and other layout options. You can use `Flex` and `Grid` together to build different
 parts of your application, and even nest them to create more complex layouts.
 
-In general, you should prefer using flex and grid over other CSS layout models. Spacing between components
-should be left to parent layout components rather than added as via margins to specific children. This ensures
-ensure components are composable when reused in different places, and ensures that spacing is consistent.
+You should prefer using flex and grid over other CSS layout models. Spacing between components
+should be left to parent layout components rather than added directly to the children. This ensures components are composable when reused in different places, and ensures that spacing is consistent.
```

Comment:
```suggestion
You should prefer using flex and grid over other CSS layout models. Spacing between components should be managed by parent layout components rather than added directly to the children. This ensures components are composable when reused in different places and that spacing remains consistent.
```